src/core/a-entity.js

```diff
@@ -461,37 +495,60 @@ var proto = Object.create(ANode.prototype, {
    */
   setAttribute: {
     value: function (attr, value, componentPropValue) {
+      var debug = this.sceneEl && this.sceneEl.getAttribute('debug');
```

Comment:
Maybe `isDebugMode`. In case we ever have debug console logs `debug('this is a log')`